packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox-group/example.html

```diff
@@ -0,0 +1,180 @@
+<h1>gux-form-field-checkbox-group</h1>
+
+<p>
+  slot a input[type=checkbox] element into a gux-form-field-checkbox element
+</p>
```

Comment:
Should this be `slot gux-form-field-checkbox elements into a gux-form-field-checkbox-group component`